Description: cmake files are installed in the wrong location which make them
 unusable and broken.
Bug-Debian: http://bugs.debian.org/1024772
Author: Dylan Aïssi <daissi@debian.org>
Forwarded: not-needed
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -325,9 +325,9 @@
 GET_FILENAME_COMPONENT(CONFIG_FILE_PATH ${CMAKE_CURRENT_BINARY_DIR}/cpuinfo-config.cmake ABSOLUTE)
 CONFIGURE_PACKAGE_CONFIG_FILE(
   cmake/cpuinfo-config.cmake.in ${CONFIG_FILE_PATH}
-  INSTALL_DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME})
+  INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 INSTALL(FILES ${CONFIG_FILE_PATH}
-  DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}) # cpuinfo_DIR ${prefix}/share/cpuinfo
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
 INSTALL(TARGETS cpuinfo
   EXPORT cpuinfo-targets
@@ -337,7 +337,7 @@
 
 INSTALL(EXPORT cpuinfo-targets
   NAMESPACE ${PROJECT_NAME}:: # IMPORTED cpuinfo::cpuinfo
-  DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME})
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
 # ---[ cpuinfo micro-benchmarks
 IF(CPUINFO_SUPPORTED_PLATFORM AND CPUINFO_BUILD_BENCHMARKS)
